lib/dpl/provider/pages.rb

```diff
@@ -17,6 +17,7 @@ class Pages < Provider
         - project-name [optional, defaults to fqdn or repo slug]
         - email [optional, defaults to deploy@travis-ci.org]
         - name [optional, defaults to Deployment Bot]
+        - deployment-file [optional, defaults to false]
```

Comment:
Please also add this to README

Reply:
Optional, defaults to false, enables the creation of deployment-info file?